cirq-core/cirq/circuits/circuit.py

```diff
@@ -1880,10 +1880,11 @@ def earliest_available_moment(
         while k > 0:
             k -= 1
             moment = self._moments[k]
+            if moment.operates_on(op_qubits):
+                return last_available
```

Comment:
Should this `break` instead, since we already have `return last_available` after the loop? Same this for the following if-statement.

Reply:
Functionally the same, but I think returning reads easier since I don't have to go down and see what the rest of the function does as I'm reading it.